exspy/models/edssemmodel.py

```diff
@@ -33,13 +33,31 @@ class EDSSEMModel(EDSModel):
     auto_background : bool
         If True, adds automatically a polynomial order 6 to the model,
         using the edsmodel.add_polynomial_background method.
+    xray_line_source : str, default 'xraydb'
```

Comment:
This docstring is correct but not the one for `EDSModel`. Is it worth defining the docstring in https://github.com/hyperspy/exspy/blob/main/exspy/_docstrings/model.py and re-using it for both?